app/Repositories/BaseRepository.php

```diff
@@ -26,14 +26,53 @@ public function all()
     }
 
     /**
-     * Creating a new resource
+     * [create description]
```

Comment:
[rank A] what's it means? Please try to comment for function clearly as much as good.
(check for other your comments, too)